strings/language/german.txt

```diff
@@ -0,0 +1,180 @@
+ah@=ach
+all@=alles
+amazing@=ausgezeichnet
+and@=und
+angel@=engel
+answer@=antwort
+apple@=apfel
+attention@=achtung
+backpack@=rucksack
+bed@=bett
+beer@=bier
+best@=beste
+beutiful@=schoen
+bible@=bibel
+blood@=blut
+bomb@=bombe
+book@=buch
+books@=buecher
+bread@=brot
+broke@=brechte
+broken@=kaputt
+bullshit@=quatsch
+burger@=berger
+canon@=kanone
+capitalism@=kapitalismus
+captain@=kapitaen
+car@=auto
+cars@=autos
+cargo@=freight
+cat@=katze
+cats@=katzen
+cheers@=prost
+cheese@=kaese
+child@=kind
+children@=kinder
+christmas@=weinachts
+cigar@=zigarre
+clear@=klar
+coffee@=kaffee
+cold@=kalt
+cow@=kuh
+day@=tag
+days@=tage
+delicious@=lecker
+device@=ding
+do@=tu
+doctor@=doktor
+doctors@=doktors
+does@=tut
+dog@=hound
+dogs@=hounds
+doing@=tun
+doodad@=ding
+door@=tuer
+drink@=trink
+electricity@=elektrizitaet
+emperor@=kaiser
+end@=ende
+exactly@=genau
+fat@=fett
+father@=vater
+fire@=feuer
+five@=fuenf
+flash@=blitz
+four@=vier
+free@=frei
+freeway@=autobahn
+fries@=pommes
+full@=voll
+gadget@=ding
+girl@=maedchen
+god@=gott
+god@=gotte
```

Comment:
```suggestion
gods@=gotter
```

god defined twice? cant find much reference to gotte. assume you meant gods/gotter?

Reply:
I was typing fast- i did mean gotter haha